src/reformat/alpha.rs

```diff
@@ -12,7 +12,7 @@ impl rgb::Image {
         if self.pixels().is_null() || self.row_bytes == 0 {
             return Err(AvifError::ReformatFailed);
         }
-        if !self.has_alpha() || self.alpha_premultiplied {
+        if !self.has_alpha() {
             return Err(AvifError::InvalidArgument);
         }
         match libyuv::process_alpha(self, true) {
```

Comment:
(cannot add a comment two lines below...)

If `alpha_premultiplied` is an input parameter, we should not do `self.alpha_premultiplied = true;`.